analytics/models.py

```diff
@@ -112,3 +113,32 @@ class CalendarExport(models.Model):
     time_created = models.DateTimeField(auto_now_add=True)
     school = models.CharField(max_length=50)
     is_google_calendar = models.BooleanField(blank=True, default=False)
+
+
+class ErrorLog(models.Model):
+    """
+    Logs errors that occur on the site.
+    """
+
+    time_occurred = models.DateTimeField(auto_now_add=True)
+    message = models.CharField(max_length=1000)
+    name = models.CharField(max_length=100)
+    stack = models.TextField()
```

Comment:
what is this field for?

Reply:
It basically stores the error stack: where the error orginates and how it proporgates through the application.
This is used for debugging purposes